src/UniversalApp.js

```diff
@@ -55,7 +56,7 @@ const initServer = function(app, routes, apolloServerOptions, production) {
                 : context
           })
             .then(html => {
-              res.status(200);
+              res.status(responseStatusCode);
```

Comment:
How about rename as `statusCode` ?

Reply:
Is `statusCode` clear enough? (The status code of routing?)

Reply:
I think it's clear enough.
What will be misunderstood with this?

Reply:
Actually I am not quite sure what does the status code of a route mean?

Reply:
Should be a route stateless?

Reply:
After searching other resource, I think you are right. `responseStatusCode` may be more specific!

Reply:
Thanks for you feedback. Good discussion 👍